index.html

```diff
@@ -4,7 +4,356 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Surbhi Kukreti" />
+    <meta name="description" content="HalloCart Home page" />
+    <link rel="stylesheet" href="/css/common.css" />
+
+    <!-- <link rel="stylesheet" href="http://127.0.0.1:5501/halloUi-share.css" /> -->
+    <link
+      rel="stylesheet"
+      href="https://halloui.netlify.app/halloUi-share.css"
+    />
+
+    <script src="/carousal.js" defer></script>
+    <script
+      src="https://kit.fontawesome.com/c780d21494.js"
+      crossorigin="anonymous"
```

Comment:
font awesome can be used with link tag. Is there any reason of using scripts ?

Reply:
i made a kit and it gave me this script tag so i used it

Reply:
Please go through the below link and find exact version of cdn you've used in your project and use it and please verify once the all the font awesome related changes stays the same or different.
https://cdnjs.com/libraries/font-awesome